.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "airbnb-base"
```

Comment:
you're going to need to add:
```
"settings": {
    "import/core-modules": ["electron-packager"]
}
```
To avoid an `eslint` error when testing your specs

Reply:
The result of eslint seems good right now. Could you open it a little bit more? Why would we need to add that?

Reply:
I have other projects where `eslint` will complain that `electron-packager` (in this case) is listed under `devDependencies` instead of `dependencies` because it's explicitly being used for the specs. By setting this preference, `eslint` knows to ignore that rule for that module.
If `eslint` is not complaining about it, then it's fine, I guess you don't need it.

Reply:
Also, this line has 4 spaces as indentation rather than 2, which you're using everywhere else.